packages/api/src/controllers/tags.controller.ts

```diff
@@ -27,4 +29,11 @@ export class TagsController {
         const createTagReq = await useValidate(createTagSchema, body);
         return this.tagsService.create(createTagReq);
     }
+
+    @UseGuards(JwtAuthGuard, CanAccessResourceGuard)
+    @Patch("/users/:id/tags")
+    async update(@Body() body: UpdateTagNameRequest): Promise<TransactionTag> {
+        const updateTagsReq = await useValidate(updateTagNameSchema, body);
+        return await this.tagsService.update(updateTagsReq);
```

Comment:
Don't await on the return when you have a return type of promise

```suggestion
        return this.tagsService.update(updateTagsReq);
```